### PR TITLE
BITA-464 Add error code 0369 "Invalid device id"

### PR DESCRIPTION
**Pull request by viccomx**
Required error for correct handling of devices.

PHP PR - https://github.com/bitsoex/bitsoex/pull/4109

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -200,6 +200,7 @@ error categories, the last two digits define specific errors.
 * 0366: Two factor authentication method is not enabled
 * 0367: Unable to generate quote at the present time
 * 0368: Rejected withdrawal to restricted address
+* 0369: Incorrect device id (non-existent or does not belong to the user)
```
Comment (domix) [inline source/index.html.md:203]: Not sure if it's good idea to use the same error code for different reasons. For final users I think it's better to have a fine grained error code/description.

Comment (viccomx) [reply source/index.html.md:203]: Not sure I'm following, error code `0359` is new, while the description is from my point of view very descriptive. Can you please expand on the idea?

Comment (domix) [reply source/index.html.md:203]: looks like the message is for either 'an incorrect device id' **OR** 'does not belong to the user'

Comment (thenewversion) [reply source/index.html.md:203]: I'd leave it as just `incorrect device id` then

Comment (viccomx) [reply source/index.html.md]: @domix @thenewversion - thanks for your input. Changes addressed.

**Review by domix — APPROVED** [review]

**Review by alejandro-martin — APPROVED** [review]

**Review by thenewversion — APPROVED** [review]

**Pull request merged**